src/components/DocumentationTopic/Summary.vue

```diff
@@ -27,8 +27,10 @@ export default {
 .summary {
   color: var(--colors-secondary-label, var(--color-secondary-label));
   font-size: rem(16px);
```

Comment:
Can we use `font-styles` mixin instead of keeping hardcoded values? I know that you didn't added them but it would be a good opportunity to clean those.